h2o-extensions/target-encoder/build.gradle

```diff
@@ -9,6 +9,8 @@ dependencies {
         
     testCompile "junit:junit:${junitVersion}"
     testCompile "org.mockito:mockito-core:2.23.0"
+    testCompile "com.pholser:junit-quickcheck-core:0.9"
```

Comment:
Decided to add it within this PR but might consider to add this dependency separately if there will be concerns about it

Reply:
Not an issue - the usage is an issue, see bellow